boot/environ/lib/firmware/efi/firmware.c

```diff
@@ -280,7 +280,7 @@ EfiVmOpenProtocol (
     }
 
     /* Check what address the interface lives at, and translate it */
-    InterfaceVa = (PVOID)InterfaceAddress.LowPart;
+    InterfaceVa = PhysicalAddressToPtr(InterfaceAddress);
```

Comment:
I'm a bit surprised by this change (and the followings). Up to know, we were using the LowPart as pointer. With your change, we're now using the full 64 bits as address. It is properly initialized on 32 bits platform?

Reply:
LOL, I hope so. And if if that is not the case, it is a bug. I seriously hope nobody would be that stupid to deliberately leave the upper part uninitialized. That would be like using an uninitialized OBJECT_ATTRIBUTES structure to pass around a UNICODE_STRING.